Sources/SpelledPitch/PitchSpeller/Wetherfield/WeightedGraphSchemeProtocol.swift

```diff
@@ -21,25 +23,70 @@ extension WeightedGraphSchemeProtocol {
         H: WeightedGraphSchemeProtocol,
         H.Weight == Weight
     {
-        return H.init { self.weight(Edge(f($0.a),f($0.b))) }
+        return H { self.weight(from: f($0.a), to: f($0.b)) }
     }
-}
 
-extension WeightedGraphSchemeProtocol {
     @inlinable
     func unweighted <H> () -> H where
         H: UnweightedGraphSchemeProtocol,
         H.Edge == Edge
     {
-        return H.init { self.weight($0) != nil }
+        return H { self.weight($0) != nil }
+    }
+}
+
+extension WeightedGraphSchemeProtocol {
+    
+    /// - Returns: rhs weight function whenever the lhs is missing the edge in question.
+    /// This function is non-commutative.
+    static func + (lhs: Self, rhs: Self) -> Self {
+        return Self { edge in
+            lhs.weight(edge) ?? rhs.weight(edge)
```

Comment:
Are we supposed to be adding here if they are both present?

Reply:
I don't want to add in cases where they are both non-`nil`... throw an error?

Reply:
This looks great, then!